modules/authorization/proc-defining-authorizations-in-external-files-by-using-the-operator.adoc

```diff
@@ -75,9 +75,9 @@ $ oc create configmap rbac-policies \
      --from-file=rbac-conditional-policies.yaml
 ----
 
-. Update your {product-short} `Backstage` custom resource to mount in the {product-short} filesystem your files from the `rbac-policies` config map:
+. Update your `{product-custom-resource-type}` link:{configuring-book-url}[custom resource] to mount in the {product-short} filesystem your files from the `rbac-policies` config map:
```

Comment:
The link is a really nice idea. It points to the guide explaining how to configure your custom `Backstage` custom resource, rather than to a generic doc about custom resources. Therefore, consider this change:

```suggestion
. Update link:{configuring-book-url}[your `{product-custom-resource-type}` custom resource] to mount in the {product-short} filesystem your files from the `rbac-policies` config map:
```